README.md

```diff
@@ -44,8 +44,8 @@ Any changes to the local version of Html-tools will trigger a `make`.
 
 For production usage, remember to
 
-1. Build with `jsmin` and `csscompressor` installed through `pip`
-2. Enable gzip compression on your server
+1. Build with `jsmin` and `csscompressor` installed through `pip`.
+1. Enable gzip compression on your server.
```

Comment:
Why?

Reply:
We've had this conversation before. It makes diffs smaller if you need to insert something.